packages/client/src/components/app/Button.svelte

```diff
@@ -16,6 +18,16 @@
   export let icon = null
   export let active = false
 
+  const handleOnClick = async () => {
+    if (handlingOnClick) {
+      return
+    }
+
+    handlingOnClick = true
+    await onClick()
```

Comment:
When there are no actions specified, `onClick` will be `null` so I'm fairly sure this will throw an error. We should check if it's null, or add in optional chaining.